share/models/validators.py

```diff
@@ -118,7 +118,7 @@ def json_schema_for_field(self, field):
             return rel
         if field.choices:
             return {
-                'enum': field.choices,
+                'enum': [c[0] for c in field.choices],
```

Comment:
I thought choices [had to be](https://docs.djangoproject.com/en/1.9/ref/models/fields/#choices) a list of 2-tuples, which breaks validation because `'key' != ('key', 'label')`. If there were choices fields that were lists of strings (or grouped choices, come to think of it), it will break.